cores/arduino/WSerial.h

```diff
@@ -67,45 +67,29 @@
   #endif /* !HWSERIAL_NONE && SERIAL_UART_INSTANCE */
 
 
-  #if defined(ENABLE_HWSERIAL1)
```

Comment:
Is it necessary to remove the definition of ENABLE_HWSERIALx here for it to take effect?

Reply:
OK, maybe this is a “too fast” workaround,... it would be best to edit all the “variant_CH32xxx.h” files and change the definitions in them '#define SERIAL_UART_INSTANCE 1' to '#define SERIAL_UART_INSTANCE “n”' where “n” is the number of available U(S)ART for a given MCU.